datasources/src/main/scala/com/normation/plugins/datasources/QueryService.scala

```diff
@@ -244,7 +244,7 @@ class HttpQueryDataSourceService(
     for {
       mode          <- globalPolicyMode()
       updated       <- tasks(nodes, info.policyServers, mode, info.parameters).timeout(timeout).provide(clock).notOptional(
-                         s"Timout error after ${timeout.asScala.toString()}"
+                         s"Timeout error after ${timeout.asScala.toString()}"
```

Comment:
use `.render` in place of `toString` to have an human readable format

Reply:
thanks!